app/views/devise/sessions/_mfa_enrolment.erb

```diff
@@ -1,29 +1,32 @@
-<h1 class="govuk-heading-xl"><%= t('mfa_enrolment.heading') %></h1>
-
 <div class="govuk-grid-row">
-  <div class="govuk-grid-column-one-half">
-
-      <%= f.label :totp_code, class: "govuk-label" %>
-      <%= f.text_field :totp_code, autofocus: true, autocomplete: 'off', class: "govuk-input govuk-input--width-10" %>
-      <%= f.hidden_field :email, value: { email: false } %>
-
-  </div>
-
-  <div class="govuk-grid-column-one-half">
-    <div class="mfa-qr-code">
-      <%= raw(@secret_code_svg) %>
-    </div>
+  <div class="govuk-grid-column-full">
+    <h1 class="govuk-heading-xl"><%= t('mfa_enrolment.heading') %></h1>
+      <p class="govuk-body-m"><%= t('mfa_enrolment.description') %></p>
+        <div style="text-align:left;">
```

Comment:
never use inline styles... it shouldn't be neccessary anyway, it should align left automatically...